### ISSUE TITLE
The results vary on each run.

### ISSUE THREAD

**Issue by mengweiwei99** [flatironinstitute/mountainsort5#10]
Hello! I have observed that when using program 2, the results vary even when keeping all parameters unchanged. Specifically, the number of detected spikes and the quantity of clustered units differ between each run. The raw data recordings typically span 5 minutes with a sampling rate of 25,000. The parameters I have configured are as follows:
sorting = ss.run_sorter(sorter_name="mountainsort5", recording=recording_preprocessed,
                            output_folder=output_folder,
                            detect_threshold=5, detect_time_radius_msec=1.5,
                            snippet_mask_radius=150,
                            scheme2_phase1_detect_channel_radius=150,
                            scheme2_training_duration_sec=60)

**Comment by magland** [comment]
Yes, the clustering algorithm used by mountainsort is non-deterministic.
